test/Microsoft.Tye.Extensions.Configuration.Tests/TyeConfigurationExtensionsTest.cs

```diff
@@ -104,21 +104,21 @@ public void GetConnectionString_WithBinding_DoesNotCombineHostAndPort()
         }
 
         [Fact]
-        public void GetServiceUri_WithoutBinding_GetsConnectionStringIfSet()
+        public void GetServiceUri_WithoutBinding_IgnoresConnectionStringIfSet()
         {
             var builder = new ConfigurationBuilder();
             builder.AddInMemoryCollection(new Dictionary<string, string>()
             {
-                { "connectionstrings:myservice", "http://expected.example.com" },
-                { "service:myservice:protocol", "https" },
-                { "service:myservice:host", "example.com" },
-                { "service:myservice:port", "443" },
+                { "connectionstrings:myservice", "https://example.com" },
+                { "service:myservice:protocol", "http" },
+                { "service:myservice:host", "expected.example.com" },
+                { "service:myservice:port", "80" },
             });
 
             var configuration = builder.Build();
 
             var result = configuration.GetServiceUri("myservice");
-            Assert.Equal(new Uri("http://expected.example.com"), result);
+            Assert.Equal(new Uri("http://expected.example.com:80"), result);

```

Comment:
Why append 80? Should we have smarts around 443 and 80?

Reply:
we could, none of the code we have today does this.

Reply:
This doesn't actually matter. URI will deal with this for us. If you look at teh codez, you can see that it's wrapped in `new Uri`. I'll change the test.